Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/InternalExposureNotificationClient.kt

```diff
@@ -160,4 +160,11 @@ object InternalExposureNotificationClient {
                     cont.resumeWithException(it)
                 }
         }
+
+    /**
+     * Indicates if device supports scanning without location service
+     *
+     * @return
+     */
+    fun deviceSupportsLocationlessScanning() = exposureNotificationClient.deviceSupportsLocationlessScanning()
```

Comment:
what happens if the api is not available on this device yet? this could lead to issues if uncaught.

Reply:
even if it doensn't seem so, the method is robust.
Keep in mind that this method is not returning a task or something.
It will return a bool result even the api is not available.